### ISSUE TITLE
New enhanced fork of python-podgen that is being updated and improved!

### ISSUE THREAD

**Issue by gabriel4649** [tobinus/python-podgen#122]
It seems like @tobinus is no longer active in this project, so we created a new fork with some enhancements! 

It's called [po2gen](https://gitlab.com/caproni-podcast-publishing/pod2gen), check it out.

**Comment by tobinus** [comment]
Ah, cool. For the record, I intend to continue with PodGen at some point, I just haven't had the capacity to do so.

I hope you don't mind if I close this issue whenever I do return, since it is not an actionable issue.

**Comment by gabrieljoel** [comment]
Yeah, of course you may close it! I'm glad to hear you are back. 

We should collaborate! Please checkout https://gitlab.com/caproni-podcast-publishing/pod2gen. We've made many changes on top of python-podgen. Maybe you want to merge in these changes into python-podgen.

**Comment by dreamflasher** [comment]
@gabriel4649 @tobinus Currently, neither project seems to be being maintained? At least both didn't have any commits the last 2 years…
